Components/Image/image.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Image</title>
+    <link rel="stylesheet" href="/Components/Image/image.css">
+    <link rel="stylesheet" href="/docs.css">
+    <script src="https://kit.fontawesome.com/2a3afcb5fd.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <header class="header">
+        <i class="fas fa-fighter-jet header-icon"></i>
+        <h1 class="header-text"></i>QuickUI</h1>
+        <a class="header-link" class="header-link" href="/index.html">Home</a>
+    </header>
+    <nav>
+        <ul class="navigation">
+            <li class="nav-link"><a class="nav-anchor" href="/Components/installation.html">Installation</a></li>
+            <li class="nav-link"><a class="nav-anchor" href="/Components/Avatar/avatar.html">Avatar</a></li>
+            <li class="nav-link"><a class="nav-anchor" href="/Components/Alert/alert.html">Alert</a></li>
+            <li class="nav-link"><a class="nav-anchor" href="/Components/Badge/badge.html">Badges</a></li>
+            <li class="nav-link"><a class="nav-anchor" href="/Components/Button/button.html">Button</a></li>
+            <li class="nav-link"><a class="nav-anchor" href="/Components/Card/card.html">Card</a></li>
+            <li class="nav-link"><a class="nav-anchor active" href="/Components/Image/image.html">Image</a></li>
+            <li class="nav-link"><a class="nav-anchor" href="/Components/Input/input.html">Input</a></li>
+        </ul>
+    </nav>
+
+    <section class="main-container">
+        <h1 class="main-heading">Image</h1>
+        <div class="separator"></div>
+        <h3 class="sub-heading">Responsive Image</h3>
+   <p class="des-para">Add img-responsive class to img tag. Image will grow to the 100% width of it's parent container.</p>
```

Comment:
Maybe take an image that covers the entire container.
Right now, a scroll bar is appearing mostly due to the iframe not being responsive.
![Screenshot (532)](https://user-images.githubusercontent.com/44032674/152388340-ffdcb6a7-919d-4f44-8d72-3532ec19071f.png)

Reply:
Yahh will make iframe responsive then I think issue will solve

Reply:
Made that image responsive have a look